newscript.go

```diff
@@ -0,0 +1,405 @@
+	err := cmd8.Run()
+	if err != nil {
+		fmt.Printf("didn't clone Nichrome")
+		return err
+	}
+	cmd9 := exec.Command("cp", "NiChrome/CONFIG", fmt.Sprintf("%s/.config", linuxVersion))
+	err = cmd9.Run()
+	if err != nil {
+		fmt.Println("3")
+		return err
+	}
+	cmd10 := exec.Command("cp", "/tmp/initramfs.linux_amd64.cpio", fmt.Sprintf("%s/", linuxVersion))
+	err = cmd10.Run()
+	if err != nil {
+		fmt.Println("2")
+		return err
+	}
+	fmt.Printf("pwd before make is %s\n", workingDir)	
+	cmd11 := exec.Command("make", "--directory", linuxVersion, "-j64")
+	err = cmd11.Run()
+	if err != nil {
+		fmt.Println("4")
+		return err
+	}
+	if _, err := os.Stat(filepath.Join(linuxVersion, "/arch/x86/boot/bzImage")); err != nil {
+		return err
+	}
+	fmt.Printf("bzImage created")
+	return nil
+}
+
+
+
+// TODO: add the chroot to github
+func findVbutil() error{
+	path, err := exec.LookPath("futility")
+	if err != nil {
+		log.Fatal("Make the chromium package to access Futility")
+	}
+	fmt.Printf("futility is available at %s\n", path)
+	return nil
+}
+
+func buildVbutil() error{
+	fmt.Printf("-------- Building in Vbutil\n")
+	cmd8 := exec.Command("git", "clone", "https://chromium.googlesource.com/chromiumos/platform/vboot_reference")
+	err := cmd8.Run()
+	if err != nil {
+		fmt.Printf("didn't get chromium repo")
+		return err
+	}
+	return nil
+	cmd9 := exec.Command("git", "checkout", "3f3a496a23088731e4ab5654b02fbc13a6881c65")
+	err = cmd9.Run()
+	if err != nil {
+		fmt.Printf("couldn't checkout the right branch")
+		return err
+	}
+	return nil
+	cmd10 := exec.Command("make", "-j64")
+	err = cmd10.Run()
+	if err != nil {
+		fmt.Printf("Make failed. Please try to manually install")
+		return err
+	}
+	return nil
+
+}
+
+func vbutilIt() error {
+	fmt.Printf("-------- VBUTILING\n")
+	buildVbutil()
+	findVbutil()
+	fmt.Printf("-------- VBUTILING  contd. \n")
+	newKern := "newKern"
+	if err := ioutil.WriteFile("config.txt", []byte("loglevel=7"), 0777); err != nil {
+		return err
+	}
+	if err := ioutil.WriteFile("nocontent.efi", []byte("no content"), 0777); err != nil {
+		return err
+	}
+	bzImage := fmt.Sprintf("%s/arch/x86/boot/bzImage",linuxVersion)
+	fmt.Printf("Bz image is located at %s \n", bzImage)
+	cmd14 := exec.Command("futility", "vbutil_kernel",  "--pack", newKern, "--keyblock", "/usr/share/vboot/devkeys/kernel.keyblock", "--signprivate", "/usr/share/vboot/devkeys/kernel_data_key.vbprivk", "--version", "1", "--vmlinuz", "linux-4.12.7/arch/x86/boot/bzImage" , "--bootloader", "nocontent.efi", "--config", "config.txt",  "--arch", "x86")	
+	stdoutStderr, err := cmd14.CombinedOutput()	
+	fmt.Printf("%s\n", stdoutStderr)
+	if err != nil {
+		fmt.Printf("\ny\n")
+		return err
+	}
+	dd()
+	return nil
+}
+
+func dd() error{
+	var location = "/dev/sda2"
+	for true {
+		fmt.Printf("Where do you want to put this kernel (/dev/sda2?)")	
+		_, err := fmt.Scanf("%s",&location)
+		if err != nil {
+			return err	
+		}
+		if _, err = os.Stat(location); err != nil{
```

Comment:
later, we can look at the returned info to make sure it's ok. For example, if they enter 
/etc/passwd
since you run this as sudo, they're going to destroy /etc/passwd.